classes/class-ledyer-fields.php

```diff
@@ -216,6 +216,29 @@ public static function fields() {
 			),
 		);
 
+		$devmode = 'local' === wp_get_environment_type() || 'development' === wp_get_environment_type();
```

Comment:
Just a note on settings in general, not on these settings specifically.

If urls.terms are not set in settings, there will be a silent crash.

Then in the logs, the following can be found:

`{"400":[" settings.urls.terms has wrong type"]}`

which is Ledyer's response